### ISSUE TITLE
xypad broken

### ISSUE THREAD

**Issue by Madsim** [birkir/react-three-gui#8]
The xypad seems to be broken. It won't work in my code, nor in the example.

**Comment by stolinski** [comment]
I'm seeing the same thing with a basic

```
  const { x, y } = useControl("Rotation", {
    type: "xypad",
  });
```

**Comment by creativiii** [comment]
Hey, so I played around with the source code for a bit and it turns out it's not actually broken.

You just need to add the option `scrub: true`, and it will work.

Maybe that should be more clear or `scrub` should always be true when using XYpad? Would love to hear from @birkir, I can create a pull request if you don't have time yourself. Let me know.

**Comment by birkir** [comment]
I will look at this!

**Comment by birkir** [comment]
Should be fixed with 0.2.0

react-spring changed the `onFrame` api so it should work now with `onChange`
